### PR TITLE
Admin css / bootstrap regressions

### PR DESCRIPTION
**Pull request by deanmarcussen**
Related to https://github.com/OrchardCMS/OrchardCore/issues/6615

To demonstrate a little what is going on

### CODE REVIEW

**Comment by deanmarcussen** [comment]
This fixes regressions from #6509 and returns the admin css sizing to as before.

It also fixes a regression from moving to bootstrap 4 (from v3) where col-md had no meaning in bs3, but does in bs4.
Our css was using a custom `col-md` to indicate a medium size column, which then was affecting bs4 `col-md` which has a meaning in bs4.

To resolve this, I have refactored the editor views using `col-md` and `col-sm`, expecting a medium or small size column to use a better named class `col-medium` and `col-small` where they previously used `col-md`.

This returns all admin editor views to the previous sizing conventions.

It does not, however, totally fix the issue that in some places we use a `col-medium` and others we use things like `col-md-6` but I figure we should address that separately, as @hishamco has started doing in his pr.

@hishamco this should not interfere with your pr, but will cause some conflicts. I suspect you didn't see it because maybe you need to refresh your browser cache.

**Comment by hishamco** [comment]
@deanmarcussen shall I wait for your PR to merge, or just refresh my browser cache?

File: src/OrchardCore.Modules/OrchardCore.Deployment/Views/Items/RecipeFileDeploymentStep.Fields.Edit.cshtml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <div class="form-group">
     <div class="row">
-        <div class="col-sm">
+        <div class="all">
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Deployment/Views/Items/RecipeFileDeploymentStep.Fields.Edit.cshtml:7]: Just I have a doubt about this one, or the intent to take the full width? If is that, why I didn't see `all` in SCSS file?

Also we could have a better meaningful name

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.Deployment/Views/Items/RecipeFileDeploymentStep.Fields.Edit.cshtml]: yeah good catch, typo

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Deployment/Views/Items/RecipeFileDeploymentStep.Fields.Edit.cshtml]: It happens ;)

**Pull request merged**